docs/reference/api/connectivity/bluetooth/Gap.md

```diff
@@ -26,6 +26,20 @@ Privacy is a feature that allows a device to avoid being tracked by other (untru
 
 You need to enable privacy by calling `enablePrivacy()` after initializing the SecurityManager because privacy requires SecurityManager to handle IRKs. Set the behavior of privacy enabled devices by using `setCentralPrivacyConfiguration()`, which specifies what the device should be with devices using random addresses, and `setPeripheralPrivacyConfiguration`. Random addresses that privacy enabled devices generate can be of two types: resolvable (by devices who have the IRK) and unresolvable. You can't use unresolvable addresses for connecting and connectable advertising; therefore, use a resolvable one for these, regardless of the privacy configuration.
 
+#### Modulation schemes
+
+When supported by the host and controller, you can select different modulation schemes:
+
+ - LE 1M PHY.
+ - LE 2M PHY.
+ - LE coded PHY.
+ 
+These provide different compromises between bandwidth, power usage and error resiliency (see BLUETOOTH SPECIFICATION Version 5.0 Vol 1, Part A - 1.2).
```

Comment:
Query: Do we have a link to this doc?

Reply:
As far as I know there is no online version, you have to get the PDF from the bluetooth website. This is not required reading, this is just helpful to the inquisitive minds.